README.md

```diff
@@ -153,19 +153,19 @@ A `Traversable` instance can be lifted as a `Query`. There are two main usages f
 **contains**
 
 ```scala
-def find(radiusList: List[Float]) = quote {
```

Comment:
I think we should keep the examples using `List`. Using `Seq` is definitely a bad practice since the implementation can be mutable.